db/seeds.rb

```diff
@@ -45,10 +45,15 @@
 end
 
 Logger.new(STDOUT).info 'Start features seed'
+Feature.create(name: 'volunteer_ops_create')
```

Comment:
it makes sense for these create things to here - I wonder if we removed them for a purpose.  I know we concluded that we should avoid data being generated in migrations which was where they were originally ...